document_sftp/models/document_sftp_root.py

```diff
@@ -59,6 +63,18 @@ def _open(self, path, flags, attr):
         raise NotImplementedError()
 
     @api.model
+    def _remove(self, path):
+        """Return file attributes"""
+        raise NotImplementedError()
```

Comment:
good question. I don't really have an opinion about this, the good thing about the exception is that it's quite clear what's going on, whereas a standard return value might also point to a permission issue or the like